### PR TITLE
Add loads account test to master

### PR DESCRIPTION
**Pull request by Call3z**
Please add test

### CODE REVIEW

**Comment by MattiasNackademin** [comment]
Please add a comment to your test

File: XUnitTest/TrustorDbTests.cs [hunk 1]
```diff
@@ -33,5 +33,13 @@ public void TrustorDb_Loads_Customers()
 
             Assert.Equal(expectedOutput,actual);
         }
+
+        [Fact]
```
Comment (Call3z) [inline XUnitTest/TrustorDbTests.cs:37]: Tests the loading of accounts

Comment (MattiasNackademin) [reply XUnitTest/TrustorDbTests.cs]: Good!!!

**Comment by MattiasNackademin** [comment]
I will merge

**Pull request merged**